src/hooks/hooks.test.ts

```diff
@@ -34,3 +34,20 @@ describe('useTabs', () => {
     });
   });
 });
+
+describe('useDateInput', () => {
+  it('returns correct props', () => {
+    // REF https://stackoverflow.com/questions/45644098/testing-anonymous-function-equality-with-jest
+    const setStateMock = expect.any(Function);
+    const { result } = renderHook(() => useDateInput());
+
+    expect(result.current).toEqual({
```

Comment:
This check doesn't do anything useful
We need to do something like this `jest.spyOn(React, 'useState').mockImplementation(...)`
And return some specific dateValues object
We need two cases for this, so we can check that:
1. If we specify all day/month/year - isFilled will be true, resultDate is calculated
2. If we specify only day - isFilled will be false, isPartiallyEmpty will be true, resultDate is not ready